client/coral-embed-stream/src/containers/Stream.js

```diff
@@ -112,7 +112,8 @@ class StreamContainer extends React.Component {
 
   componentDidMount() {
     if (this.props.previousTab) {
-      this.props.data.refetch();
+      this.props.data.refetch()
+        .then(() => this.props.setCommentCountCache(0));
```

Comment:
Hm. Why do you set it to `0`? I'm not sure if that works. I think it should be set to `asset.commentCount` from the new query result. Do you have access to it inside this callback?